packages/health/lib/src/health_factory.dart

```diff
@@ -325,19 +347,25 @@ class HealthFactory {
     DateTime startTime,
     DateTime endTime,
     List<HealthDataType> types,
+    bool isPriorityQueue,
```

Comment:
maybe for better clarity change to named params?

Reply:
Sure why not